business_requirement_deliverable_project/views/business_view.xml

```diff
@@ -41,7 +41,7 @@
                             type="object"
                             class="oe_highlight"
                             groups="business_requirement.group_business_requirement_manager"
-                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'approved')]}"/>
+                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'stakeholder_approval')]}"/>
```

Comment:
@elicoidal if we apply the change of the name as in https://github.com/sudhir-serpentcs/business-requirement/pull/9 also to this module for V8, then the fix above would be fine, right?